heudiconv/tests/test_bids.py

```diff
@@ -1116,3 +1126,32 @@ def test_BIDSFile():
     # -for an existing entity, you can overwrite it with "set":
     my_bids_file.set('echo', '2')
     assert my_bids_file['echo'] == '2'
+
+
+@pytest.mark.skipif(not have_datalad, reason="no datalad")
+def test_ME_mag_phase_conversion(tmpdir, subID='MEGRE', heuristic='bids_ME'):
+    """ Unit test for the case of multi-echo GRE data with
+    magnitude and phase.
+    The different echoes should be labeled automatically.
+    """
+    tmpdir.chdir()
+    tmppath = tmpdir.strpath
+    try:
+        datadir = fetch_data(tmppath, "dicoms/velasco/{subID}")
+    except IncompleteResultsError as exc:
+        pytest.skip("Failed to fetch test data: %s" % str(exc))
```

Comment:
in datalad we trust? ;)

```suggestion
    datadir = fetch_data(tmppath, f"dicoms/velasco/{subID}")
```

Reply:
my bad -- apparently (for some reason, didn't check) this is a common usage pattern, so let's keep as that for now, I will just fix the f-string

Reply:
Sorry, I'm not really sure if I need to do anything on this PR...

Reply:
CI failures still to be checked, test fixed

Reply:
I forgot to add the extension for the heuristic file in `tests.test_bids.test_ME_mag_phase_conversion`.
Addressed in https://github.com/cbinyu/heudiconv/commit/aee3d9ba81f55067a79de3ffcfb3bd39d74e6659.
It should be ready now.